dispatch/api/serializers.py

```diff
@@ -410,8 +425,8 @@ class ArticleSerializer(DispatchModelSerializer, DispatchPublishableSerializer):
 
     content = ContentSerializer()
 
-    authors = PersonSerializer(many=True, read_only=True)
-    author_ids = serializers.ListField(write_only=True, child=serializers.IntegerField())
+    authors = AuthorSerializer(source='get_authors', many=True, read_only=True)
+    author_ids = serializers.ListField(write_only=True, child=serializers.JSONField())
```

Comment:
Once `AuthorValidator` is defined, you can include it here in the `validators` argument